mailet/api/pom.xml

```diff
@@ -55,11 +55,16 @@
             <version>1.7.5</version>
```

Comment:
By the way if `ŋetMessage().addHeader()` invalidate the parsing I think we have no choice to recompute it all the time no?